components/event/EventCard/EventCard.module.css

```diff
@@ -0,0 +1,248 @@
+.container {
```

Comment:
Lots of styles in this file are copied from other events file, remove erroneous @claalmve